drivers/network/dd/netkvm/Common/ParaNdis-VirtIO.c

```diff
@@ -368,22 +368,22 @@ static void vdev_sleep(void *context, unsigned int msecs)
 }
 
 VirtIOSystemOps ParaNdisSystemOps = {
-    .vdev_read_byte = ReadVirtIODeviceByte,
```

Comment:
Regarding the designaters: you can still do something like this for documentation purposes:
```c
    /* .vdev_read_byte = */ ReadVirtIODeviceByte,
```
etc.

Reply:
Thanks, I fixed.